### ISSUE TITLE
Upgrading from 4.1.7 to 4.1.8

### ISSUE THREAD

**Issue by jenspeters** [irods/irods#3210]
Hi,

I've got an issue upgrading on centos 6 from 4.1.7 to 4.1.8.
Fails with: 
```
sudo rpm -U irods-icat-4.1.8-centos6-x86_64.rpm
Upgrading Existing iRODS Installation
Configuration Schema Version is already up to date (version=2).
Stopping iRODS server...
Traceback (most recent call last):
  File "/var/lib/irods/packaging/update_catalog_schema.py", line 212, in <module>
    main()
  File "/var/lib/irods/packaging/update_catalog_schema.py", line 208, in main
    update_database_to_latest_version()
  File "/var/lib/irods/packaging/update_catalog_schema.py", line 163, in update_database_to_latest_version
    current_schema_version = get_current_schema_version(cfg)
  File "/var/lib/irods/packaging/update_catalog_schema.py", line 96, in get_current_schema_version
    'get_current_schema_version: failed to parse schema_version\n\n' + '\n'.join(sql_output_lines))
RuntimeError: get_current_schema_version: failed to parse schema_version
warning: %post(irods-icat-4.1.8-0.x86_64) scriptlet failed, exit status 1
```
Kind regards,

Jens

**Comment by jenspeters** [comment]
Seems nothing to do with the update itself. The 4.1.7 Server couldn't start before, too. 
```
./irods start
Starting iRODS server...
Traceback (most recent call last):
  File "/var/lib/irods/iRODS/scripts/python/get_db_schema_version.py", line 77, in <module>
    current_schema_version = get_current_schema_version(cfg)
  File "/var/lib/irods/iRODS/scripts/python/get_db_schema_version.py", line 61, in get_current_schema_version
    'get_current_schema_version: failed to find result line for schema_version\n\n{}'.format(format_cmd_result(result)))
ValueError: zero length field name in format
Confirming catalog_schema_version... Success
    Skipped JSON Configuration Validation [off]
iRODS server failed to start.
```

**Comment by jenspeters** [comment]
Could be some Proxy issue?

**Comment by trel** [comment]
What happens if you turn the validation back on before starting the 4.1.7 server?

**Comment by jenspeters** [comment]
Ok, found it, but error Messages could be improved. The database was not listening correctly. 

https://groups.google.com/forum/#!topic/irod-chat/eeXujAj2wnc

**Comment by trel** [comment]
Glad you found it.

The error messages from the control scripts have been much improved and these changes will be included in 4.2.
